recipes/boost/all/conanfile.py

```diff
@@ -141,7 +141,7 @@ def configure(self):
                     raise ConanInvalidConfiguration("Boost '%s' library requires multi threading" % lib)
 
     def build_requirements(self):
-        self.build_requires("b2/4.2.0")
+        self.build_requires("b2/4.3.0")
```

Comment:
minor change: could you move `build_requirements` method between `package_id` and `source` methods? (to follow `conan install`'s executions order. It makes recipes easier to understand... at least for me :p )

Reply:
Sure, no problems